### PR TITLE
Namespace changed, all tests passing.

### PR DESCRIPTION
**Pull request by brentschmaltz**
moved System.IdentityModel.Tokens -> Microsoft.IdentityModel.Tokens

### CODE REVIEW

File: test/System.IdentityModel.Tokens.Jwt.Tests/CreateAndValidateTokens.cs [hunk 1]
```diff
@@ -25,13 +25,18 @@
...
 using System.Collections.Generic;
 using System.Globalization;
-using System.IdentityModel.Tokens.Tests;
 using System.Security.Claims;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.IdentityModel.Tokens.Tests;
 using Xunit;
 
+// since we are in the System ns, we need to map to M.IM.Tokens
+using SigningCreds = Microsoft.IdentityModel.Tokens.SigningCredentials;
```
Comment (tushargupta51) [inline test/System.IdentityModel.Tokens.Jwt.Tests/CreateAndValidateTokens.cs]: @brentschmaltz  Why is this needed?

Comment (brentschmaltz) [inline test/System.IdentityModel.Tokens.Jwt.Tests/CreateAndValidateTokens.cs]: As I hoped the comment indicated, the compiler will default to System.IdentityModel.Tokens.SigningCredentials. So we have to explicitly define the type.

File: test/System.IdentityModel.Tokens.Jwt.Tests/project.json [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
+        "Microsoft.IdentityModel.Tokens.Tests": "5.0.0-*"
     },
     "commands": {
         "test": "xunit.runner.aspnet"
     },
     "frameworks": {
-        "dnx451": { },
+        "dnx451": {
+          "frameworkAssemblies": {
+            "System.IdentityModel": "4.0.0.0"
```
Comment (tushargupta51) [inline test/System.IdentityModel.Tokens.Jwt.Tests/project.json]: Why adding this dependency? I don't think we are using any types in this assembly.

Comment (brentschmaltz) [inline test/System.IdentityModel.Tokens.Jwt.Tests/project.json]: The is testing that S.IM can be included. Which was not possible before.

**Comment by tushargupta51** [comment]
:shipit:  it has cla-required label for some reason
